Bottom: 43549522befa451661f22b1496073e501345f44b
Top:    fbb689bc3e5fc4dc670b312a486301677a0617aa
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 13:45:19 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index be5e7f7..a78428a 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1147,17 +1147,17 @@ static unsigned long getnumopt(char *str, char option)
 
 static const char *getstropt(char *str, char option)
 {
-	char *names = str;
-	char *val = NULL;
-	char *end;
+	char *names;
+	char *val;
 
 	if (!str || !*str)
 		return NULL;
 
-	if (*str == '=')
-		str++;
+	names = str;
+	if (*names == '=')
+		names++;
 
-	val = strdup(str);
+	val = strdup(names);
 	if (!val) {
 		fprintf(stderr,
 			"%s: option -%c, couldn't allocate storage for argument\n",
